specs/opcode/16AND_17OR_18XOR.md

```diff
@@ -0,0 +1,39 @@
+# AND, OR, XOR op code
+
+## Procedure
+
+Pop two EVM words `a` and `b` from the stack, and the output `c` is pushed to
+the stack.
+
+`a`, `b`, and `c` are all EVM words. We break three EVM words into 32 bytes and
+apply the lookup to the 32 chunks of `a`, `b`, and `c` to see if
+`a[i] OP b[i] == c[i]` holds for `i = 0..31`, where `OP` belongs to
```

Comment:
If using rust's range style, it should be `32` or `=31`, right?
```suggestion
`a[i] OP b[i] == c[i]` holds for `i = 0..32`, where `OP` belongs to
```

Reply:
updated